src/js/common/components/form-elements/ErrorableCheckboxes.jsx

```diff
@@ -0,0 +1,204 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash';
+import classNames from 'classnames';
+
+import ToolTip from './ToolTip';
+import ExpandingGroup from '../../../common/components/form-elements/ExpandingGroup';
+
+import { makeField } from '../../model/fields.js';
+
+/**
+ * A checkbox button group with a label.
+ *
+ * Validation has the following props.
+
+ * `additionalFieldsetClass` - String for any additional fieldset classes.
+ * `additionalLegendClass` - String for any additional legend classes.
+ * `label` - String for the group field label.
+ * `name` - String for the name attribute.
+ * `toolTipText` - String with help text for user.
+ * `tabIndex` - Number for keyboard tab order.
+ * `options` - Array of options to populate group.
+ * `required` - is this field required.
+ * `values` - object. Values of the checkbox field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableCheckboxes extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+    this.getMatchingSubSection = this.getMatchingSubSection.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = this.props.id || _.uniqueId('errorable-checkbox-buttons-');
+  }
+
+  getMatchingSubSection(checked, optionValues) {
+    if (checked && this.props.children) {
+      const children = _.isArray(this.props.children) ? this.props.children : [this.props.children];
```

Comment:
A quick check of the inspector indicates that it can be an array:
![screen shot 2018-04-02 at 5 18 52 pm](https://user-images.githubusercontent.com/16051172/38222509-f93da244-3699-11e8-8802-89125e522419.png)
The [docs](https://reactjs.org/docs/react-api.html#reactchildren) explain that this isn't always the case however.